configuration/cdap-site.xml

```diff
@@ -900,34 +948,21 @@
   </property>
 
   <property>
-    <name>log.saver.run.memory.megs</name>
+    <name>log.saver.container.memory.mb</name>
```

Comment:
Update `themes.json` names for these changes, or you'll break the widgets in the UI.